src/FsCheck/Gen.fs

```diff
@@ -330,14 +343,38 @@ module Gen =
     ///Sequence the given list of generators into a generator of a list.
     //[category: Creating generators from generators]
     [<CompiledName("Sequence"); CompilerMessage("This method is not intended for use from F#.", 10001, IsHidden=true, IsError=false)>]
-    let sequenceToSeq generators = 
-        generators |> sequence |> map List.toSeq
+    let sequenceToSeq<'T, 'TSource when 'TSource :> IEnumerable<Gen<'T>>> (generators: 'TSource) =
```

Comment:
Ah, didn't think about that - I'll revert this part of the changes. Now that I think about it again, I'm not sure adding the extra type parameter was going to accomplish what I had in mind anyway.